Vic2ToHoI4/Source/HOI4World/WarCreator/HoI4WarCreator.cpp

```diff
@@ -13,7 +13,7 @@
 
 
 
-HoI4WarCreator::HoI4WarCreator(const HoI4::World* world, const HoI4::MapData& theMapData):
+HoI4WarCreator::HoI4WarCreator(HoI4::World* world, const HoI4::MapData& theMapData):
```

Comment:
why world raw pointer? Unless you'll be fixing that later.

Reply:
Because war creator, sadly. But one day it will be fixed!